ui/src/app/pages/data/concept/concept-homepage.spec.tsx

```diff
@@ -41,46 +40,20 @@ describe('ConceptHomepage', () => {
     currentWorkspaceStore.next(workspaceDataStub);
   });
 
-  it('should render', () => {
-    const wrapper = component();
-    expect(wrapper).toBeTruthy();
+  it('should render', async () => {
+    component();
+    await screen.findByPlaceholderText(/search concepts in domain/i);
   });
 
   it('should have one card per domain.', async () => {
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.find('[data-test-id="domain-box-name"]').length).toBe(
-      DomainStubVariables.STUB_DOMAINS.length
-    );
+    component();
+    const domainBoxes = await screen.findAllByTestId('domain-box');
+    expect(domainBoxes.length).toBe(DomainStubVariables.STUB_DOMAINS.length);
   });
 
   it('should have one card per survey.', async () => {
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.find('[data-test-id="survey-box-name"]').length).toBe(
-      SurveyStubVariables.STUB_SURVEYS.length
-    );
+    component();
+    const surveyBoxes = await screen.findAllByTestId('survey-box');
+    expect(surveyBoxes.length).toBe(SurveyStubVariables.STUB_SURVEYS.length);
   });
-
-  // Need to update with new api calls
```

Comment:
let's check with Data Apps on whether this is OK to remove

Reply:
Good call. Looks like we are good to delete.
https://pmi-engteam.slack.com/archives/C0189PPPKFF/p1715625349342469